udata_front/theme/gouvfr/templates/header.html

```diff
@@ -29,7 +29,10 @@
                             {% include theme('svg/marianne.svg') %}
                         </a>
                     </div>
-                    <nav class="col hidden visible-xl mobile-menu row-inline justify-end self-start vuejs">
+                    <nav
+                            class="col hidden visible-xl mobile-menu row-inline justify-end self-start vuejs"
+                            role="navigation"
+                    >
                         <a href="/datasets" title="{{ _('Search for data') }}" class="mr-md"
```

Comment:
Shouldn't there be a `role="search"` here? Shouldn't the `role="navigation"` be on the menu part only?

Reply:
After discussion, it seems that since hidden elements are removed from the accessibility tree, we can have the navigation and search roles twice for desktop and responsive.